# Required Entries
string entry_name
# Data
bool data_bool
string data_str
uint32 data_uint32
int32 data_int32
float32 data_float
geometry_msgs/Quaternion data_quat
geometry_msgs/Vector3 data_vec3
---
bool success